### DISCUSSION TITLE
Did you mean "aghi"?

### DISCUSSION THREAD

**Discussion by FezVrasta** [FiloSottile/age#329] [General]
I'm trying to understand the meaning of the library name, sorry if I'm missing something.

The link on the README points to Google Translate that incorrectly translates "aghe" as "needles", while actually the translation of "needles" to italian is "aghi".

Alternative actual words that end in "aghe" are:

- saghe
- maghe
- daghe
- braghe
- draghe
- piaghe
- vaghe

Source | https://www.listediparole.it/f/a/4/parolefinisconoaghe.htm

Could you clarify please?

**Comment by alerque** [comment]
I don't think the Italian *meaning* has anything in particular to do with this, the issue is pronunciation. The release message should probably read "pronounced like the Italian" instead of just "like the Italian" to clarify Does that answer your question?

**Comment by FezVrasta** [comment]
Thanks, that makes sense, maybe remove the Google Translate link then?

I'm not sure a lot of people will understand the pronunciation with the current approach, but that's definitely up to you.

**Comment by roychri** [comment]
I do not think they should remove the link as it helps to understand how to pronounce it. It was clear to me that it was to show pronounciation and not that it  was the "meaning".

**Comment by FezVrasta** [comment]
My tone of voice is quite irritating but feel free to use this recording if you want an italian pronunciation of the name.

https://vocaroo.com/1cX3WbMGAps0

**Comment by FiloSottile** [comment]
Indeed, the point is not the translation, but the pronunciation. age is pronounced like the hypothetical "aghe" Italian word would be. (Italian has fairly rigid rules for spelling and pronunciation. We don't do spelling bees.)

I'll replace the link with something that only plays audio.
